src/styles/tags.css

```diff
@@ -1,21 +1,53 @@
 .tag-list {
+  line-height: 1.7;
   list-style: none;
   padding: 0;
 }
 
-.tag-list li {
+.tag-list.inline-flex {
```

Comment:
Hold on, I thought this was just a part of the tag-list class. If not, then it should be, and the `inline-flex` class shouldnt exist